### DISCUSSION TITLE
Add WYSIWYG editor in Flows email operation

### DISCUSSION THREAD

**Discussion by izoukhai** [directus/directus#14363] [Feature Requests]
For the moment, Flows allow us to send emails using MD but it's quite limited in features. It would be great to have the choice between MD and WYSIWYG.

**Comment by rijkvanzanten** [comment]
Isn't Markdown HTML compliant as well? Not as nice of an editing experience, but you should be able to "just" paste in some HTML and it should work

**Reply by izoukhai** [reply]
I thought the same thing but HTML tags aren't rendered in the email. The preview mode show them correctly though

**Comment by izoukhai** [comment]
I think the problem comes from here: https://github.com/directus/directus/blob/2607037ffe08ad7e2c423d6a8fab67cb94600beb/api/src/services/notifications.ts#L43-L52

See here, Directus applies the 'base' template on every emails, except for the ones coming from Flows operation: 
https://github.com/directus/directus/blob/857eae54defc166bef6198d482bdaba2a0b876cf/api/src/operations/mail/index.ts#L17-L21

**Comment by Nisgrak** [comment]
The best option maybe is allow to use the emails templates, the md function sanitizes the html so any <style> will be removed.

**Reply by izoukhai** [reply]
Yea but templates are stored locally, maybe we could implement a new table that holds all the templates like this:

`
id: number;
name: string;
variables: {type: 'number' | 'string' | 'json', key: string}
`

It could lead to a whole new feature like an email template builder using [Unlayed Editor](https://unlayer.com/embed) or custom HTML, who knows.

**Reply by rijkvanzanten** [reply]
@izoukhai Templates can be extended through the extensions folder though https://docs.directus.io/extensions/email-templates/, so adding a way to enter a template string ID might still make sense 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
